Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -3992,6 +3992,8 @@ - (void)collectionViewDidChangeLayout
 
             ConversationViewItem *_Nullable viewItem = self.viewItemMap[interaction.uniqueId];
             if (viewItem) {
+                // It's possible the interaction has been updated.
+                [viewItem replaceInteraction:interaction transaction:transaction];
```

Comment:
See the existing usage of `reloadInteractionForViewItem` which should already handle this case.